.jenkins/scripts/solutions/init-config.sh

```diff
@@ -0,0 +1,4 @@
+echo "deb [arch=amd64] https://packages.microsoft.com/repos/azure-cli/ bionic main" | sudo tee /etc/apt/sources.list.d/azure-cli.list
```

Comment:
Regarding this file name `solutions/init-config.sh`: this file isn't initialization for solutions test per se. It is for setting up the apt repo for azure-cli, which is a dependency only due to the fact that we're using an alternative authentication method in Jenkins for some solution tests.

Reply:
good point, I'll clean up the naming